### PR TITLE
Add anchor certificate to DEP profile template

### PR DESCRIPTION
**Pull request by jessepeterson**
In support of #107

### CODE REVIEW

File: cmd/mdmctl/apply_dep_profile.go [hunk 1]
```diff
@@ -16,15 +18,23 @@ func (cmd *applyCommand) applyDEPProfile(args []string) error {
 	var (
 		flProfilePath = flagset.String("f", "", "filename of DEP profile to apply")
 		flTemplate    = flagset.Bool("template", false, "print a JSON example of a DEP profile")
+		flAnchorFile  = flagset.String("anchor", "", "filename of PEM certificate to add to trusted anchors")
```
Comment (groob) [inline cmd/mdmctl/apply_dep_profile.go]: Help text should indicate that anchor expects a valid chain.

Comment (jessepeterson) [inline cmd/mdmctl/apply_dep_profile.go]: Hmm, what do you mean a valid chain? In the case of a self-signed cert a single cert is the "valid chain." Happy to modify the text though. What do you suggest?

Comment (groob) [inline cmd/mdmctl/apply_dep_profile.go]: I can generate a self signed root CA, and then an intermediary, and then sign a cert with that intermediary. The whole chain needs to be added, no?

Comment (jessepeterson) [inline cmd/mdmctl/apply_dep_profile.go]: Hmm. Well, I haven't tested. I'd think you can just trust the leaf and that's good. I would also think you could just trust the CA (but could see how that wouldn't work, too). I know it works specifying a self-signed, though, but that doesn't address the question.

Comment (groob) [inline cmd/mdmctl/apply_dep_profile.go]: I doubt that a leaf would work in this scenario. Usually trust requires a full chain.

**Comment by jessepeterson** [comment]
Using new mdmctl DEP commands (thanks @groob!) have confirmed that generating a DEP profile template using `mdmctl apply dep-profiles -template -use-server-cert` works for specifying a single self-signed cert. DEP enrollment successfully completes for macOS. I'm sure that'll be good enough for most folks. But I'd like to test using self-signed CA with intermediates. The code doesn't currently support having more than one certificate added per each method (either `-anchor` or `-user-server-cert`) so that could be a problem if that doesn't work and folks are using something like that.

**Pull request merged**